### ISSUE TITLE
Disk usage

### ISSUE THREAD

**Issue by jmartty** [fpw/avitab#59]
Would it be possible to do disk cleanup of the downloaded tiles after X amount of time passed since filemtime ?

**Comment by fpw** [comment]
Good idea. I need to do some testing since this is very OS and even file system specific, but I'll see what I can do.
